src/Service/CodeDeploy/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Change Log
 
+## NOT RELEASED
+
 ## 0.2.0
 
 ### Deprecation
```

Comment:
I think "deprecation" is just for thing we added deprecation notices about. We should use `Removed` here. 

```suggestion
### Removed
```